docs/components/alert.md

```diff
@@ -14,6 +14,38 @@
 
 The main content of the alert.
 
+### alertSecondaryMessage
```

Comment:
Instead of adding another prop, could we just change KAlert to support line breaks in the `alertMessage` prop?

Reply:
Alternatively, couldn't this be resolved by just slotting the message instead of putting the message in the prop?